softdevice_controller/include/sdc.h

```diff
@@ -222,7 +222,6 @@ extern "C" {
  * @param[in] max_tx_data_size The maximum size of data which can be sent.
  * @param[in] failure_reporting_enabled Whether failure reporting is enabled.
  */
-#ifdef SDC_USE_NEW_MEM_API
 #define __MEM_MINIMAL_PERIODIC_ADV_RSP_SET_SIZE_WITH_RX (465)
 #define __MEM_MINIMAL_PERIODIC_ADV_RSP_SET_SIZE_WITHOUT_RX (160)
 #define SDC_MEM_PER_PERIODIC_ADV_RSP_SET(max_adv_data, tx_buffer_count, rx_buffer_count, max_tx_data_size, failure_reporting_enabled) \
```

Comment:
should probably get the latest sdc.h. Some small doc changes requested by Sergey.

Reply:
the doc changes should be included.